Sources/Commands/PackageCommands/Init.swift

```diff
@@ -42,17 +42,9 @@ extension SwiftPackageCommand {
                 """))
         var initMode: InitPackage.PackageType = .library
 
-        /// Whether to enable support for XCTest.
-        @Flag(name: .customLong("xctest"),
-              inversion: .prefixedEnableDisable,
-              help: "Enable support for XCTest")
-        var enableXCTestSupport: Bool = true
-
-        /// Whether to enable support for swift-testing.
-        @Flag(name: .customLong("experimental-swift-testing"),
-              inversion: .prefixedEnableDisable,
-              help: "Enable experimental support for swift-testing")
-        var enableSwiftTestingLibrarySupport: Bool = false
+        /// Which testing libraries to use (and any related options.)
+        @OptionGroup()
+        var testingLibraryOptions: TestLibraryOptions
```

Comment:
there's a naming inconsistency here, which can make it harder to use textual search when looking for relevant properties or types, one way to resolve it could be this

```suggestion
        var testLibraryOptions: TestLibraryOptions
```

Reply:
I will rename it.